### PR TITLE
Update UG and DG

### PR DESCRIPTION
**Pull request by chesterhow**
UG
- Update portions of UG
- Update screenshots

DG
- Update manual testing portion for `add-date`, `del-date`, `details` and `view` command

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -612,12 +544,49 @@ The first person on the list is born before 12-12-2020.
 
 ### Deleting a special date: `del-date`
 
+Prerequisites: List all person using the `list` command. There is at least a person present in the list. 
+The first person on the list has at least one date.
+
+1. Deleting a date from an existing person
+
+   1. Test case: `del-date 1 i/1` <br>
```
Comment (lue97) [inline docs/DeveloperGuide.md:552]: perhaps you can specify that the first date is deleted?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -124,14 +123,14 @@ The Streaks dashboard shows how consistent you are in meeting your friends up ti
 
 #### Full Details of a Person
 As a person in the FriendDex can contain a lot of information, not all of it is displayed in the main list of persons.
-Upon execution of the `details` command, FriendDex will display the full details of the specified person on the details panel.
+Upon execution of the `details` command, FriendDex will display the full details of the specified person on the details panel. See [Details Command](l#viewing-full-details--details) for more details.
```
Comment (lue97) [inline docs/UserGuide.md]: broken link

**Review by lue97 — APPROVED** [review]
LGTM with minor errors

**Review by lue97 — APPROVED** [review]

File: docs/UserGuide.md [hunk 3]
```diff
@@ -12,23 +12,22 @@ A good example is students who are struggling to do well in both their studies a
...
+Furthermore, the guide gives information on the User Interface (UI) and the other useful features of FriendDex. Each section of the guide can be read independently.
+For an outline of the content in this document, refer to the Table of Contents above. You may also use your document viewer’s Find function to quickly navigate to the content you want to know more about.
 
-It is generally advised for new users to at least read through the [Quick Start](#quick-start) section to familiarise themselves with FriendDex.
+It is generally advisable for new users to visit the [Quick Start](#quick-start) section first to familiarise themselves with FriendDex.
 
-Note the following symbols and formatting used in this document:
+### Symbols
 
-`list n/Close Friends` <br>
```
Comment (ivantjh) [inline docs/UserGuide.md:30]: Can add the markup portion here for CS2101? We wrote this here because it was in the sample UG

Comment (chesterhow) [reply docs/UserGuide.md:30]: Okay but need to reword. The current description is inconsistent as just one section later we use it for "Java `11`".

**Pull request merged**